source/grid/manifold_lib.cc

```diff
@@ -2026,7 +2027,7 @@ TransfiniteInterpolationManifold<dim, spacedim>::pull_back(
           return chart_point + update;
         }
 
-      // every 8 iterations, including the first time around, we create an
+      // every 9 iterations, including the first time around, we create an
```

Comment:
I hope the success does not depend on whether we do 8 or 9 iterations. :-)

Reply:
No, but I observed it was faster on average... :smile: